controller/Controller.php

```diff
@@ -62,4 +62,45 @@ function tutorsAjax()
         }
 
     }
+
+    /**
+     * Ajax logic for checklist page
+     * @author Oleg
+     */
+    function checklistAjax(){
+        $this->_db->updateYearData($_POST['column'],$_POST['value'],$_POST['year']);
+    }
+
+    /**
+     * View of all the required forms. Lets the user check of or uncheck a form if it has been
+     * completed
+     * @param $param int is the id of the user
```

Comment:
We've been doing @param datatype $varibable. Either is PEAR but it should be consistent